store/store.go

```diff
@@ -1366,6 +1366,7 @@ var download = func(ctx context.Context, name, sha3_384, downloadURL string, use
 		return err
 	}
 
+	var fatalErr error
```

Comment:
The error is not actually fatal, since we can retry and it may go away. `finalErr` or `resultErr` might be better.